### DISCUSSION TITLE
prior define upper and lower on y

### DISCUSSION THREAD

**Discussion by olesendan** [lindeloev/mcp#67] [Q&A]
Hi.

I'm trying to learn, how to use your package.

I'm messing around with priors, and would like to set bounds on y. Is this possible?
I think i can set cp_i = "T(MINX, MAXX)". Is it possible to do something like cp_i = T(y >= 1, y<=2)

Greetings from denmark
Dan Olesen

**Comment by lindeloev** [comment]
Hi Dan,

Sorry for missing your issue! Priors only apply to inferred parameters. If y is observed, it has no prior. `cp_[some-number-here]` is the parameter representing the (inferred) change point location. Say you wanted to put a bounded normal prior on the first change point:

```r
mcp(segments, data, prior = list(cp_1 = "dnorm(10, 3) T(MINX, 20)"))
```

If you really want "bounds" on y, maybe that's a preprocessing step where you filter away "bad" values, like `y = y[y > 0 & y < 10]`. Maybe I'm answering the wrong question. If so, could you be a bit more specific?
